azure-pipelines.yml

```diff
@@ -54,6 +54,9 @@ jobs:
         - openssl-devel
         - freetype-devel
         - libpng-devel
+        - libffi-devel
+        - python-devel
```

Comment:
Why python-devel?

Reply:
It's what cryptography said to pre-install in order to compile the library.

Reply:
Could you stick a comment above in the file saying that's why they're here? And is it possible to install just for 32bit instead of all jobs?

Reply:
It is only installed for the 32bit builds.